pykafka/test/utils.py

```diff
@@ -2,6 +2,16 @@
 
 from pykafka.test.kafka_instance import KafkaInstance, KafkaConnection
 
+try:
+    import unittest2 as unittest
```

Comment:
Yeah, I realized that afterwards.  Originally I was going to try to make everything in py26 work since it was in tox but decided that the support wasn't important to me :P